examples/src/main/java/com/jagrosh/jdautilities/examples/command/GuildlistCommand.java

```diff
@@ -47,35 +53,80 @@
 )
 @RequiredPermissions({Permission.MESSAGE_EMBED_LINKS, Permission.MESSAGE_ADD_REACTION})
 @Author("John Grosh (jagrosh)")
-public class GuildlistCommand extends Command {
+public class GuildlistCommand extends Command
+{
+
+    private final EventWaiter waiter;
 
-    private final Paginator.Builder pbuilder;
     public GuildlistCommand(EventWaiter waiter)
     {
         this.name = "guildlist";
+        this.aliases = new String[]{"listguilds", "guilds"};
         this.help = "shows the list of guilds the bot is on";
         this.arguments = "[pagenum]";
-        this.botPermissions = new Permission[]{Permission.MESSAGE_EMBED_LINKS, Permission.MESSAGE_ADD_REACTION};
         this.guildOnly = false;
         this.ownerCommand = true;
-        pbuilder = new Paginator.Builder().setColumns(1)
-                .setItemsPerPage(10)
-                .showPageNumbers(true)
-                .waitOnSinglePage(false)
-                .useNumberedItems(false)
-                .setFinalAction(m -> {
-                    try {
-                        m.clearReactions().queue();
-                    } catch(PermissionException ex) {
-                        m.delete().queue();
-                    }
-                })
-                .setEventWaiter(waiter)
-                .setTimeout(1, TimeUnit.MINUTES);
+        this.botPermissions = new Permission[]{
+            Permission.MESSAGE_EMBED_LINKS, Permission.MESSAGE_ADD_REACTION
+        };
+
+        this.waiter = waiter;
+    }
+
+    /** @return a new Paginator */
+    private Paginator.Builder getBuilder(String text)
+    {
+        Checks.notNull(text, "text");
+        return new Paginator.Builder().setItemsPerPage(10)
+                            .showPageNumbers(true).waitOnSinglePage(false)
+                            .useNumberedItems(true).setEventWaiter(this.waiter)
+                            .setTimeout(1, TimeUnit.MINUTES)
+                            .setText(text).setFinalAction(m -> {
+                try {
+                    m.clearReactions().queueAfter(250, TimeUnit.MILLISECONDS);
+                } catch (PermissionException ex) {
+                    m.delete().queue();
+                }
+            });
+    }
+
+    /**
+     * @param guild The guild to count
+     * @return the number of members which are not bots
+     */
+    private static int getHumanCount(Guild guild)
+    {
+        return guild.getMembers().size() - guild.getMembers().stream()
```

Comment:
Why convert to an array instead of `guild.getMembers().stream().filter(m -> m.getUser().isBot()).count()`?